recipe/meta.yaml

```diff
@@ -76,6 +79,20 @@ test:
     - test -f ${PREFIX}/include/{{ each_ncurses_header }}
     {% endfor %}
 
+    # Test for tinfo symbols.
+    {% set tinfo_symbols = [
+        "cbreak",
+        "keypad",
+        "stdscr"
+    ] %}
```

Comment:
We test the same symbols you have mentioned in the issue ( https://github.com/conda-forge/ncurses-feedstock/issues/9 ), @cel4. Not that the underscore is not included here as Linux names these differently. So, we handle that detail in the OS X tests below.